db/migrate/20230316034707_disable_inherits_tax_category_from_per_order_enterprise_fees.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class DisableInheritsTaxCategoryFromPerOrderEnterpriseFees < ActiveRecord::Migration[6.1]
+  def change
+    EnterpriseFee.where(id: enterpise_fee_ids)
+      .where(inherits_tax_category: true)
+      .update_all(inherits_tax_category: false)
```

Comment:
We set these into `false`, but it is not clear to me (from the code) which tax category will be selected for the fee, in this case. Ideally, it would be something like "None" or category associated to a zero tax rate. Do you know what happens with this migration @abdellani ?

Otherwise, let's move to test ready and test it there. But I feel this is an acceptance criteria which we should not overlook on testing: we need to assure that the migration sets:

- inherits_tax_category: true
- the selected tax category has a zero tax rate associated

The risk here is, that a random or first option is selected, which can lead to incorrect tax rate attribution to enterprise fees. 

I can't tell much more from the code, s